empathy.lic

```diff
@@ -0,0 +1,51 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathy
+=end
+
+custom_require.call(%w(common common-travel drinfomon))
+
+class Empathy
+  include DRC
+  include DRCT
+
+  def initialize
+    settings = get_settings
+    @stationary_skills_only = settings.crossing_training_stationary_skills_only
+    @hometown = settings.hometown
+
+    train_empathy
+  end
+  
+  def check_research
+    return unless @researching

```

Comment:
The goal of `check_research` is not to avoid breaking research, but rather to avoid letting it expire (i.e. issue a new `research blah` command as needed). Power walking can take a while.